From 64653bcd8c1f1deeb8483e0d3e4ee8768a807b5d Mon Sep 17 00:00:00 2001
From: Petr Mladek <pmladek@suse.cz>
Date: Mon, 18 Jul 2011 12:32:57 +0200
Subject: [PATCH] distro-install-desktop-integration: work with normal user

+ alllow to overwrite the readonly desktop files
+ create the compat soffice symlink in DESTDIR
---
 bin/distro-install-desktop-integration |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/distro-install-desktop-integration b/bin/distro-install-desktop-integration
index c00c8c6..600c54e 100755
--- a/bin/distro-install-desktop-integration
+++ b/bin/distro-install-desktop-integration
@@ -123,7 +123,7 @@ for file in *.desktop ; do
         -e "s/\($INSTALLDIRNAME\)$PRODUCTVERSION/\1/" \
         -e "s/\($PRODUCTNAME\) $PRODUCTVERSION/\1/" \
         "$file" >"$file.new"
-    mv "$file.new" "$file"
+    mv -f "$file.new" "$file"
 done
 cd -
 
@@ -160,7 +160,7 @@ add_wrapper unopkg         unopkg  ""            "unopkg"      ""
 #       you cannot include this link directly into the package
 #       For example, the Novell package mark this symlink as %ghost
 #	and update it in %post and %postun
-ln -sf $INSTALLDIR/program/soffice $PREFIXDIR/bin/soffice
+ln -sf $INSTALLDIR/program/soffice $DESTDIR$PREFIXDIR/bin/soffice
 
 # create bash completion
 mkdir -p $DESTDIR/etc/bash_completion.d
-- 
1.7.3.4

